scripts/rubymine-git-wrapper

```diff
@@ -1,8 +1,2 @@
 #!/usr/bin/env bash
-
-if [[ "$1" == commit ]] ; then
-  shift
-  exec git duet-commit "$@"
-fi
-
-exec git "$@"
+exec git "${@/%commit/duet-commit}"
```

Comment:
I think this regex definitely would match `commit`, but this side effect could be harmless because Rubymine may not use the `-m` flag?

I also wonder about best practices with having "commit" in the name of a branch. :open_hands: :thinking: